adhocracy4/modules/models.py

```diff
@@ -25,6 +31,42 @@ def settings_instance(self):
                 return getattr(self, setting)
 
     @property
+    def is_active(self):
+        return self.phase_set \
+            .active_phases()\
+            .exists()
+
+    @property
+    def is_active_in_project(self):
+        return self == self.project.active_module
+
+    @property
+    def active_phase(self):
+        return self.phase_set \
+            .active_phases() \
+            .first()
+
+    @functional.cached_property
+    def phases(self):
+        return self.phase_set.all()
+
+    @property
+    def future_phases(self):
+        phases = self.phase_set.filter(models.Q(start_date__gt=timezone.now())
+                                       | models.Q(start_date=None))
```

Comment:
Maybe the semantics of a phase without start_date is "planned, but not yet scheduled". How does `start_date__gt` handle `None`?